rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp

```diff
@@ -539,7 +539,8 @@ void Compiler::VCMPGTUW_(u32 vd, u32 va, u32 vb) {
 void Compiler::VCTSXS(u32 vd, u32 uimm5, u32 vb) {
 	auto vb_v4f32 = GetVrAsFloatVec(vb);
 	if (uimm5) {
-		vb_v4f32 = m_ir_builder->CreateFMul(vb_v4f32, m_ir_builder->CreateVectorSplat(4, ConstantFP::get(m_ir_builder->getFloatTy(), 1 << uimm5)));
+		u64 float_value = UINT64_C(1) << uimm5;
+		vb_v4f32 = m_ir_builder->CreateFMul(vb_v4f32, m_ir_builder->CreateVectorSplat(4, ConstantFP::get(m_ir_builder->getFloatTy(), reinterpret_cast<double&>(float_value))));
```

Comment:
I don't know what it does. Vector Splat creates a vector that has the same value for every component. 
I have no idea what the instruction does though. It's strange to use shift value for double...